packages/visualizations/src/components/MapPoi/MapRender.svelte

```diff
@@ -45,6 +56,122 @@
     const sourceId = `shape-source-${mapId}`;
     const layerId = `shape-layer-${mapId}`;
 
+    // Used to render tooltips on click
+    export let renderTooltip: PoiMapRenderTooltipFunction;
+    // Used to display the tooltip on a left sidebar fixed position
+    export let fixed: boolean;
+    // Used to store keyboard navigation index
+    let keyboardTooltipNavIndex = 0;
+    // Used to distinguish click and keyboard navigation
+    let isKeyboardNavigation = false;
+    // Used to track clicked Feature id
+    let activeFeatureId: string | number | undefined | null = null;
+    // Used to store maplibre popup instance
+    let clickPopup: maplibregl.Popup;
+
+    function removeTooltip() {
+        clearActiveFeature(map, sourceId, activeFeatureId);
+        clickPopup.remove();
+    }
+    // We recreate a new Popup instance for each type of tooltips to avoid event listeners errors
+    $: if (fixed) {
+        if (clickPopup) {
+            removeTooltip();
+        }
+        clickPopup = new maplibregl.Popup({
+            closeOnClick: false,
+            closeButton: false,
+            className: 'tooltip-on-click tooltip-on-click-fixed',
+        });
+    } else {
+        if (clickPopup) {
+            removeTooltip();
+        }
+        clickPopup = new maplibregl.Popup({
+            closeOnClick: false,
+            closeButton: false,
+            className: 'tooltip-on-click',
+        });
```

Comment:
I think we can just have a `className: fixed ? '...' : '....'`. It will recreate a newPopup if `fixed` changes.